### PR TITLE
create PlaceVisitModel object

### LINKED ISSUE #7: Create Place Visits Model Objects
Create java server side model objects representing Place Visit

### PR DESCRIPTION
**Pull request by cli1903**

### CODE REVIEW

**Review by anfee1 — DISMISSED** [review]
LGTM

**Review by chuckluv — DISMISSED** [review]
Can't find anything to nit

File: frontend/src/main/java/com/google/tripmeout/frontend/PlaceVisitModel.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+public abstract class PlaceVisitModel {
+  public static Builder builder() {
+    return new AutoValue_PlaceVisitModel.Builder();
+  }
+
+  public abstract String placeId();
+  public abstract String name();
+  public abstract String tripId();
+  public abstract String userMark(); // must-see, if-time, don't-care, null
+  public abstract float latitude();
```
Comment (bsidhom) [inline frontend/src/main/java/com/google/tripmeout/frontend/PlaceVisitModel.java]: Do we have specific reason to store this as a `float`? For example, does the Maps API only handle coordinates with 32-bit precision anyway? If the answer is yes and we're only storing/passing through lat/long, this should be fine. However, if we're ever mathematically transforming the coordinates down the line, we'll want to store these as `double`s for increased precision.

**Review by bsidhom — DISMISSED** [review]
Just a note about lat/long precision. We should get this right so we don't shoot ourselves in the foot down the line.

**Review by bsidhom — APPROVED** [review]

File: frontend/src/main/java/com/google/tripmeout/frontend/PlaceVisitModel.java (hunk 1 above)
Comment (cli1903) [reply frontend/src/main/java/com/google/tripmeout/frontend/PlaceVisitModel.java]: i looked into the java maps api and it does use doubles to represent latitude and longitude so I've changed it accordingly

**Review by anfee1 — APPROVED** [review]

**Pull request merged**